### PR TITLE
otgutils update - Adding LACP and LAG stats

### PR DESCRIPTION
**Pull request by octpetre**

### CODE REVIEW

**Comment by octpetre** [comment]
kind approval reminder to @openconfig/featureprofiles-maintainers 
Thanks

**Comment by octpetre** [comment]
@bstoll could you please approve

File: internal/otgutils/print.go [hunk 1]
```diff
@@ -78,3 +78,71 @@ func LogPortMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
 	out.WriteString("\n\n")
 	t.Log(out.String())
 }
+
+// LogLagMetrics is displaying otg lag stats.
```
Comment (liulk) [inline internal/otgutils/print.go:82]: Need to uppercase LAG due to Go style initialism rules.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: internal/otgutils/print.go [hunk 2]
```diff
@@ -78,3 +78,71 @@ func LogPortMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
...
+func LogLagMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
+	t.Helper()
+	var out strings.Builder
+	out.WriteString("\nOTG LAG Metrics\n")
+	for i := 1; i <= 120; i++ {
+		out.WriteString("-")
+	}
+	out.WriteString("\n")
```
Comment (liulk) [inline internal/otgutils/print.go:90]:
```suggestion
	fmt.Fprintln(&out, strings.Repeat("-", 120))
```

Like what you did on line 55.

File: internal/otgutils/print.go [hunk 3]
```diff
@@ -78,3 +78,71 @@ func LogPortMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
...
+			lag.Name(), operStatus, framesTx, framesRx, memberPortsUP,
+		))
+	}
+	fmt.Fprintln(&out, strings.Repeat("-", 120))
+	out.WriteString("\n\n")
+	t.Log(out.String())
+}
+
+// LogLacpMetrics is displaying otg lacp stats.
+func LogLacpMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
```
Comment (liulk) [inline internal/otgutils/print.go:111]: Also need to uppercase LACP.

File: internal/otgutils/print.go [hunk 4]
```diff
@@ -78,3 +78,71 @@ func LogPortMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
...
+	t.Helper()
+	var out strings.Builder
+	out.WriteString("\nOTG LACP Metrics\n")
+	for i := 1; i <= 120; i++ {
```
Comment (liulk) [inline internal/otgutils/print.go:115]: Use strings.Repeat?

**Review by liulk — CHANGES_REQUESTED** [review]

File: internal/otgutils/print.go (hunk 1 above)
Comment (octpetre) [reply internal/otgutils/print.go:82]: Done

File: internal/otgutils/print.go (hunk 3 above)
Comment (octpetre) [reply internal/otgutils/print.go:111]: Done

File: internal/otgutils/print.go (hunk 2 above)
Comment (octpetre) [reply internal/otgutils/print.go:90]: Done

**Review by liulk — APPROVED** [review]

**Pull request merged**